compose-v2/base-images/galaxy-container-base/files/common_cleanup.sh

```diff
@@ -2,8 +2,15 @@
 
```

Comment:
can/should this be a symlink to the other file?

Reply:
I think this is not possible, unfortunately :( https://stackoverflow.com/questions/31881904/docker-follow-symlink-outside-context 
I also don't like having the same file at multiple places, but I didn't find a better solution (yet).